source/androidx.media3/media3-extractor/Transforms/Metadata.xml

```diff
@@ -85,15 +85,6 @@
     Java.Lang.Object
   </attr>
   -->
-  <add-node
-    path="/api/package[@name='androidx.media3.extractor.text']/class[@name='CuesWithTimingSubtitle']"
-    >
-    <method
-      visibility="public" abstract="false" static="false" return="java.util.List&lt;androidx.media3.common.text.Cue&gt;" name="getCues"
-      deprecated="not deprecated" final="false" native="false" synchronized="false" bridge="true" synthetic="true"
-      >
-      <parameter type="long" name="count" />
-    </method>
-  </add-node>
+  <attr path="/api/package[@name='androidx.media3.extractor.text']/class[@name='CuesWithTimingSubtitle']/method[@name='getCues']" name="return">java.util.List&lt;androidx.media3.common.text.Cue&gt;</attr>
```

Comment:
Does this change actually work, or do anything?

I don't think it's going to be able to bind `java.util.List<androidx.media3.common.text.Cue>`.